### DISCUSSION TITLE
Disable IdeaVim in "New Terminal"?

### DISCUSSION THREAD

**Discussion by andrew-bolin** [JetBrains/ideavim#945] [Q&A]
When I check the "Enable New Terminal (beta)" setting in PyCharm, the terminal uses vim keybindings. I don't want that. I want my Ctrl-C to kill scripts!

Using 'old' terminal mode is fine, the terminal behaves like a standard bash terminal.

The 'new' terminal presumably has some benefits (e.g. pop-ups for history), but I personally don't want to be mucking around with INSERT/NORMAL mode and using vim key bindings to run bash commands. So, is there a way to disable IdeaVim for the New Terminal?

I found an ancient thread about disabling IdeaVim in the terminal but people there claimed that turning off a key binding (escape for jump to editor) solved their problems - that does not solve my desire to eliminate vim from my terminal!

**Comment by AlexPl292** [comment]
Hi, thank you for the report. You can track the progress here: [VIM-3543](https://youtrack.jetbrains.com/issue/VIM-3543) Disable IdeaVim in new terminal
